vizro-core/src/vizro/models/_action/_action.py

```diff
@@ -228,6 +228,7 @@ def _transformed_inputs(self) -> list[State] | dict[str, State | ControlsStates]
                 "filter_interaction": self._get_filter_interaction_states(),
             },
             "_trigger": State(*self._first_in_chain_trigger.split(".")),
+            "_vizro_controls_store": State("vizro_controls_store", "data"),
```

Comment:
```suggestion
            "_controls_store": State("vizro_controls_store", "data"),
```

I think this is probably a good convention to follow:
* we use `vizro_` prefix for the Dash components so that user's don't accidentally clash with them, like `vizro_controls_store`, `vizro_download`, etc.
* when they're built-in arguments to actions, no need for the `vizro_` prefix. If a user makes their own `controls_store` argument then it takes priority already